docs/release.md

```diff
@@ -41,6 +41,31 @@ A couple of example PRs to help:
 - https://github.com/rancher/kontainer-driver-metadata/pull/1027
 - https://github.com/rancher/kontainer-driver-metadata/pull/1084 
 
+## Prime configuration variables
+
+When preparing a new KDM release line (or advancing an existing one), update the prime-related variables used by CI/test scripts to correctly detect prime and select a prime-published agent image:
```

Comment:
Small nit: When a new KDM branch is created the `LAST_COMMUNITY_RANCHER` needs to be set in a similar way to how `maxChannelServerVersion` is used in `data.json` - Likely something along the lines `v2.13.99`, until the prime version cut off is agreed on. Otherwise this looks good!